<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="CNU326CND1" name="com.wwt.sda_mw.TestNG_SDA" tests="6" failures="2" timestamp="8 Oct 2013 19:58:32 GMT" time="31.510" errors="0">
  <testcase name="SDA_VerifyHome_04" time="0.697" classname="com.wwt.sda_mw.TestNG_SDA">
    <failure type="org.testng.TestNGException" message="
The data provider is trying to pass 1 parameters but the method com.wwt.sda_mw.TestNG_SDA#SDA_VerifyHome_04 takes 0">
      <![CDATA[org.testng.TestNGException: 
The data provider is trying to pass 1 parameters but the method com.wwt.sda_mw.TestNG_SDA#SDA_VerifyHome_04 takes 0
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1337)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1225)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:335)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:330)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- SDA_VerifyHome_04 -->
  <testcase name="SDA_AllOrders_SearchStautsOrder_09" time="6.288" classname="com.wwt.sda_mw.TestNG_SDA"/>
  <testcase name="SDA_AllOrdersSearchSite_06" time="9.522" classname="com.wwt.sda_mw.TestNG_SDA"/>
  <testcase name="SDA_AllOrders_SeachMarketOrder_08" time="8.857" classname="com.wwt.sda_mw.TestNG_SDA"/>
  <testcase name="SDA_AllOrders_SearchSalesOrder_07" time="6.091" classname="com.wwt.sda_mw.TestNG_SDA"/>
  <testcase name="SDA_VerifyAllOrders_05" time="0.055" classname="com.wwt.sda_mw.TestNG_SDA">
    <failure type="org.testng.TestNGException" message="
The data provider is trying to pass 1 parameters but the method com.wwt.sda_mw.TestNG_SDA#SDA_VerifyAllOrders_05 takes 0">
      <![CDATA[org.testng.TestNGException: 
The data provider is trying to pass 1 parameters but the method com.wwt.sda_mw.TestNG_SDA#SDA_VerifyAllOrders_05 takes 0
	at org.testng.internal.Invoker.injectParameters(Invoker.java:1337)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1225)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:335)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:330)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- SDA_VerifyAllOrders_05 -->
</testsuite> <!-- com.wwt.sda_mw.TestNG_SDA -->
